tests/test_assets.py

```diff
@@ -598,8 +598,9 @@ def test_sid_assignment(self):
         finder = AssetFinder(metadata=metadata, allow_sid_assignment=True)
 
         # Verify that Assets were built
-        play = finder.retrieve_asset_by_identifier('PLAY')
+        play = finder.lookup_generic('PLAY', datetime.now())[0]
```

Comment:
@jfkirk is there a reason we're calling `lookup_generic` here as opposed to `lookup_symbol`?